prow/cmd/deck/main.go

```diff
@@ -161,12 +160,6 @@ func (o *options) Validate() error {
 	if o.hiddenOnly && o.showHidden {
 		return errors.New("'--hidden-only' and '--show-hidden' are mutually exclusive, the first one shows only hidden job, the second one shows both hidden and non-hidden jobs")
 	}
-	if o.gcsNoAuth {
-		logrus.Warn("'--gcs-no-auth' is deprecated and is not used anymore. We always fall back to an anonymous client now, if all other options fail.")
```

Comment:
Lets add a deprecation warning to `prow/ANNOUNCEMENTS.md` and keep this for another month, I think this might be too quick, not all Prow instances update daily

Reply:
Fine for me

Reply:
I replaced the flag removal commit with another one for the deprecation announcement